jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  transform: {
+    '^.+\\.jsx?$': ['babel-jest', { configFile: './.babelrc' }],
+  },
+  moduleFileExtensions: ['js', 'jsx'],
+  moduleDirectories: ['node_modules', 'src'],
+  testEnvironment: 'jest-environment-jsdom',
```

Comment:
Add this line to this file to make your project run without webpack errors as in this picture

![Screenshot (302)](https://github.com/user-attachments/assets/1c2f4ed6-592b-4cb7-8103-a59bb2d1c4fe)

 so the app can focus on adding the test